assets/config/postcss.config.js

```diff
@@ -16,6 +16,18 @@ module.exports = {
             ]
         }),
 
+        // PurgeCSS to remove unused classes for better page speed.
+        require("@fullhuman/postcss-purgecss")({
+            // Specify the paths to all of the template files in your project
+            content: [ "./layouts/**/*.html" ],
+            // Whitelist HubSpot specific classes so they don't get removed.
+            whitelist: ["supported-cicd-platforms", ":not"],
+            whitelistPatterns: [/^fa-/, /^hs-/, /^highlight$/, /^pagination$/, /^code-/,  /^copy-/],
+            whitelistPatternsChildren: [/^hs-/, /^highlight$/, /^pagination$/, /^code-/,  /^copy-/],
+            // Extract the default tailwind classes.
+            defaultExtractor: content => content.match(/[\w-/:]+(?<!:)/g) || [],
```

Comment:
Could you add a more descriptive comment explaining what this regex is doing?

Reply:
👍